### PR TITLE
Add missing obj file to Windows SPHINCS+ AVX2 builds

### PR DESCRIPTION
**Pull request by thomwiggers**
It didn't compile on Windows.

### CODE REVIEW

**Comment by thomwiggers** [comment]
There's still some other problem in SHA256 versions.

File: crypto_sign/sphincs-sha256-128f-robust/avx2/Makefile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 LIB=libsphincs-sha256-128f-robust_avx2.a
 
 HEADERS = params.h address.h wots.h utils.h utilsx8.h sha256avx.h sha256x8.h fors.h api.h hash.h hashx8.h hash_state.h thash.h thashx8.h  sha256.h
-OBJECTS =          address.o wots.o utils.o utilsx8.o sha256avx.o sha256x8.o fors.o hash_sha256x8.o sign.o hash_sha256.o thash_sha256_robust.o hash_sha256x8.o thash_sha256_robustx8.o sha256.o
```
Comment (mkannwischer) [inline crypto_sign/sphincs-sha256-128f-robust/avx2/Makefile:6]: What's the reason for modifying the other Makefiles as well?

Comment (thomwiggers) [reply crypto_sign/sphincs-sha256-128f-robust/avx2/Makefile:6]: It had this listed twice.

Comment (mkannwischer) [reply crypto_sign/sphincs-sha256-128f-robust/avx2/Makefile:6]: ahh, right. I missed that.

**Pull request merged**